### DISCUSSION TITLE
Super note: Cannot fully cycle through block selection menu using up/down keys

### DISCUSSION THREAD

**Discussion by xthrk** [standardnotes/forum#2576] [General]
In browser and desktop app (did not tested on mobile) in a super note, after typing /, I cannot fully cycling through the available options. The selection cycles through the available options, but the selected option is not shown, after the selection left the visible area.

**Comment by myreli** [comment]
I can [replicate](https://www.loom.com/share/f7ccc20ce2ab434f8de9ff3efaef20ac) this navigating with the keyboard arrow keys, while scrolling works fine.

**Reply by xthrk** [reply]
As soon as I reach the end of the visible list and press the down-key again, the selection disappears down into the not visible area.

Last visible selection on "Current time":
![grafik](https://user-images.githubusercontent.com/116081825/218859664-ee4465f0-d618-40fe-9ae0-3a19a4576ab0.png)

pressed down-key again, selection (grey) not visible:
![grafik](https://user-images.githubusercontent.com/116081825/218860028-c380f8a7-1631-42e7-a20d-db74347e21a5.png)

The selection is not visible, but if I press the enter-key, the selection is inserted, even it is not visible. In the above example then the "Current date", because this is the next item under "Current time".
